### PR TITLE
Encoding: add koi8-ru label

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5173/)

**Comment by annevk** [comment]
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1215860.

**Comment by annevk** [comment]
I added a second commit to align with encodings.json, so we no longer keep the distinct compatibility names table. If this all looks good I'd prefer we use "rebase and merge" so the commits are kept distinct and we don't get an ugly merge commit.

File: encoding/textdecoder-labels.html [hunk 1]
```diff
@@ -10,22 +10,23 @@
     return encoding.name !== 'replacement';
   }).forEach(function(encoding) {
     encoding.labels.forEach(function(label) {
+      const textDecoderName = encoding.name.toLowerCase() // ASCII names only, so safe
```
Comment (inexorabletash) [inline encoding/textdecoder-labels.html]: nit: trailing ;

**Review by inexorabletash — COMMENTED** [review]
lgtm with one nit

**Pull request merged**